XCUITests/SearchTest.swift

```diff
@@ -218,4 +218,38 @@ class SearchTests: BaseTestCase {
         waitForExistence(app.textFields["url"], timeout: 20)
         waitForValueContains(app.textFields["url"], value: "google")
     }
+    
+    func testSearchIconOnAboutHome() {
+        waitForTabsButton()
+        navigator.performAction(Action.OpenNewTabFromTabTray)
+        waitForTabsButton()
+        
+        // Search icon is displayed.
+        waitForExistence(app.buttons["TabToolbar.stopReloadButton"])
+        XCTAssertEqual(app.buttons["TabToolbar.stopReloadButton"].label, "Search")
+        XCTAssertTrue(app.buttons["TabToolbar.stopReloadButton"].exists)
+        app.buttons["TabToolbar.stopReloadButton"].tap()
+
+        let addressBar = app.textFields["address"]
+        XCTAssertTrue(addressBar.value(forKey: "hasKeyboardFocus") as? Bool ?? false)
+        XCTAssert(app.keyboards.count > 0, "The keyboard is not shown")
+    
+        app.textFields["address"].typeText("www.google.com\n")
```

Comment:
This would not be needed if you use the action mentioned above

Reply:
The SV testcases have a manual verification of the keyboard.  I didn't think to verify the negative of this conditional.  Is there not a way to do this properly?

Reply:
To verify the keyboard is shown you can do it as you did with:
`XCTAssert(app.keyboards.count > 0, "The keyboard is not shown")
`
Then to verify that the address text field is focused, this would work:
`XCTAssertTrue(addressBar.isEnabled)`

Reply:
> To verify the keyboard is shown you can do it as you did with:
> `XCTAssert(app.keyboards.count > 0, "The keyboard is not shown") `
> Then to verify that the address text field is focused, this would work:
> `XCTAssertTrue(addressBar.isEnabled)`

When I do this:
```
XCTAssert(app.keyboards.count > 0, "The keyboard is not shown")
```

It fails even though the keyboard is displayed.

Also, when I add this:
```
XCTAssertTrue(addressBar.isEnabled)
```
It says addressBar is unresolved. Do I need to create this variable first?

Reply:
I run that code locally and was working for me...
Have you defined: `let addressBar = app.textFields["address"]`? 

You may have to add a `waitForExistence(app.textFields["address"], timeout: 3)` to wait for the keyboard to be shown.
Also if you find issues with the simulator and the keyboard, try go to Simulator -> Device -> Erase all content and settings...sometimes it fails in my end, not being shown or recognized and that usually fixes the issues.